tools/allowed-large-files

```diff
@@ -30,6 +30,8 @@ qa-tests-backend/test-images/registry-image/tomcat-files/web.xml
 roxctl/helm/derivelocalvalues/testdata/k8s-resources/*.yaml
 scale/image/fixtures/scrape.json
 scale/mocksensor/image_names.go
+sensor/tests/replay/data/central-out.bin
+sensor/tests/replay/data/trace.jsonl
```

Comment:
Naming: do we focus on testing anything particular here? If not, then maybe we could name this file using the term `safety net`?

Reply:
I've added the prefix `safety-net` 🙂